shared/src/vyxal/elements/ElementInformation.scala

```diff
@@ -2853,14 +2865,21 @@ object ElementInformation:
     ),
     AddElement(
       symbol = "∻",
-      keywords = Seq("integer-divide", "int-div", "//"),
+      keywords =
+        Seq("integer-divide", "int-div", "//", "overwrite", "vertical-merge"),
       arity = 2,
       Options(vectorises = true),
       Overload(
         name = "Integer Divide",
         args = Seq("num", "num"),
         description = "#1 // #2",
       ),
+      Overload(
+        name = "Vertical Merge",
+        args = Seq("str", "str"),
+        description =
+          "overwrite the start of the shorter string with the longer one",
```

Comment:
this is unclear: it seems like it's adding characters from the long string at the beggining of the short one, when it's doing precisely the opposite fro mthe implementation